src/components/DeploymentScheduleCard.vue

```diff
@@ -28,13 +28,14 @@
   const props = defineProps<{
     deployment: Deployment,
     deploymentSchedule: DeploymentSchedule,
+    deploymentScheduleCompatible: DeploymentScheduleCompatible,
```

Comment:
I'm not sure I understand what this prop is doing - the `DeploymentSchedule` has the `catchup` and `maxActiveRuns` values, right? Is there a reason we can't pull them from that prop? I may be misunderstanding how these are used.